cranelift/filetests/src/zkasm_codegen.rs

```diff
@@ -231,3 +235,86 @@ fn optimize_labels(code: &[&str], func_index: usize) -> Vec<String> {
     }
     lines
 }
+
+// TODO: fix same label names in different functions
+pub fn compile_clif_function(func: &Function) -> Vec<String> {
+    let flag_builder = settings::builder();
+    let isa_builder = zkasm::isa_builder("zkasm-unknown-unknown".parse().unwrap());
+    let isa = isa_builder
+        .finish(settings::Flags::new(flag_builder))
+        .unwrap();
+    let mut context = cranelift_codegen::Context::for_function(func.clone());
+    let compiled_code = context
+        .compile(isa.as_ref(), &mut Default::default())
+        .unwrap();
+    let mut code_buffer = compiled_code.code_buffer().to_vec();
+    fix_relocs(
+        &mut code_buffer,
+        &func.params,
+        compiled_code.buffer.relocs(),
+    );
+    let code = std::str::from_utf8(&code_buffer).unwrap();
+    let mut lines: Vec<String> = code.lines().map(|s| s.to_string()).collect();
+    // TODO: I believe it can be done more beautiful way
+    let mut funcname = func.name.to_string();
+    funcname.remove(0);
+    funcname.push(':');
+    let mut res = vec![funcname];
+    res.append(&mut lines);
+    res
+}
+
+// TODO: this function should be much rewrited,
+// now it works for one very basic case
```

Comment:
Could describe the one case it works for and outline how it needs to be rewritten.

Reply:
It must be rewritten before we will migrate to this infra (optimistically before end of Feb), and I think nobody except me will do this, so this comment is just a signal for reviewers to not ask replace this assembling to something better, because I will do it in next PRs